BUILD

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
Will this file be inadvertently end up being the implicit owner of any unowned source file in the root directory? Asking because in the bazel world ownership of directories is influenced by the presence of BUILD files, and I'm not sure if that is also the case in Pants.

Not sure that is a problem, just something to note in a comment maybe.